automation/src/main/java/org/wso2/testgrid/automation/executor/JMeterResultCollector.java

```diff
@@ -60,11 +63,11 @@ public void sampleOccurred(SampleEvent sampleEvent) {
                                             "\", \"Status code\": \"",
                                             result.getResponseCode().replaceAll("\"", "\\\""),
                                             "\", \"Response Message\": \"",
-                                            result.getResponseMessage().replaceAll("\"", "\\\""),
-                                            "\", \"Sampler Data\": \"",
-                                            result.getSamplerData().replaceAll("\"", "\\\""), "\"}");
+                                            result.getResponseMessage().replaceAll("\"", "\\\""));
```

Comment:
Can we show a portion of the sampler data? It is useful when the response sizes r useful. I'm thinking of something like following?

String samplerData = result.getSamplerData();
if (samplerData.size() < 5000) {
  samplerData = samplerData.substring(0, 5000) + "\n log truncated..."
}

Reply:
Fixed in 1a609df